gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/LocalRegion.java

```diff
@@ -14546,6 +14546,13 @@ public void acquirePoolMemory(long oldSize, long newSize, boolean withEntryOverH
     }
   }
 
+
+  public void relaseMemoryForInternalRegions(){
```

Comment:
As commented above. May be no need to deduct as it is actually consuming.

Reply:
See my reply above.